docs/buildah-config.md

```diff
@@ -22,10 +22,11 @@ Defaults to false.
 Note: You can also override the default value of --add-history by setting the
 BUILDAH\_HISTORY environment variable. `export BUILDAH_HISTORY=true`
 
-**--annotation** *annotation*
+**--annotation** *annotation*=*annotation*
 
 Add an image *annotation* (e.g. annotation=*annotation*) to the image manifest
 of any images which will be built using the specified container. Can be used multiple times.
+If *annotation* has a trailing `-`, then the *annotation* is removed from the config.
```

Comment:
example?

Reply:
Examples go under the example section, and not right here, correct?